### ISSUE TITLE
README for executePath doesn't match behavior in code

### ISSUE THREAD

**Issue by AndrewRayCode** [misogi/vscode-ruby-rubocop#50]
From the readme:

```
{
  // If not specified searches for 'rubocop' executable available on PATH (default and recommended)
  "ruby.rubocop.executePath": "",
```

From the code:

https://github.com/misogi/vscode-ruby-rubocop/blob/664e19bd104dd96d914c3e4d03f6f872a41830a5/src/rubocop.ts#L39

```
if (!this.path || 0 === this.path.length) {
            vscode.window.showWarningMessage('execute path is empty! please check ruby.rubocop.executePath config');
```

See also https://github.com/Microsoft/vscode/issues/26832 so now I get this warning on every single ruby tab I open.

**Comment by AndrewRayCode** [comment]
If I do `which rubocop` and set the path to that directory, I now get the warnings in every tab:

`env: ruby_executable_hooks: No such file or directory`

**Comment by g5codyswartz** [comment]
This plugin was working for me previously and I didn't have to configure anything in the settings. I've already tried the same exact things as you did @AndrewRayCode with the same outcomes.

**Comment by aried3r** [comment]
I'm experiencing the same. It's looking for a rubocop in a Ruby version I don't have installed anymore but I was also unable to find any settings regarding the path to rubocop, I have none. Setting it in the same fashion as @AndrewRayCode I was also unable to get it working again :\

**Comment by AndrewRayCode** [comment]
I updated this plugin to the latest version and I'm getting rubocop linting again properly in files. I still have an empty executePath string. Not sure what's going on or why, since [the latest rubocop.ts is still supposed to error if there is no path specified](https://github.com/misogi/vscode-ruby-rubocop/blob/46abea79336920b44c356c7c6dfdafb500622f92/src/rubocop.ts#L46-L49)

**Comment by AndrewRayCode** [comment]
Something is wrong with this plugin that I can't figure out. Even when I manually specify the executePath it seems to be using the wrong version of rubocop. I get warnings from an invalid .rubocop.yml in every ruby file I open, but running it at the command line (with the same absolute path) shows no warnings, even when manually pointing to the config file.

I did discover that your executePath has to end in a trailing slash because this repo doesn't correctly use path.join, BUT

I would suggest using the main "ruby" vscode extension instead of this plugin, which supports rubocop, and worked out of the box.

**Comment by Thadeu** [comment]
+1

**Comment by micahlisonbee** [comment]
+1

Looks like it's caching the reference to an older version of Ruby for me as well.

**Comment by trejo08** [comment]
if you exec this command from your shell:
`user@your_computer:$~/which rubocop`

that commando will return something like this:
`/Users/your_user/.rvm/gems/ruby-2.3.1@global/bin/rubocop`

you could copy that path and paste in your vscode config file as the value of the key that mentioned above:
```
{
  // If not specified searches for 'rubocop' executable available on PATH (default and recommended)
  "ruby.rubocop.executePath": "/Users/your_user/.rvm/gems/ruby-2.3.1@global/bin/",
}
```

ensure that you have installed rubocop in the specific gemset that you are putting in the `vscode` config file

by the example I used RVM, I have installed `ruby 2.3.1` and I'm using the `global` gemset

**Comment by ctsstc** [comment]
I have ran `which rubocop`

I get: `/Users/codyswartz/.rvm/gems/ruby-2.6.3/bin/rubocop`

So I set the Execute Path to: `/Users/codyswartz/.rvm/gems/ruby-2.6.3/bin/`

But then I get the following error when I save an `rb` file:

```
env: ruby_executable_hooks: No such file or directory
```

Upon further investigation if I run the following manually from my repository's directory it is fine:

```
> /Users/codyswartz/.rvm/gems/ruby-2.6.3/bin/rubocop
```

But if I choose a random folder and run the following then I get the same error.
```
> /Users/codyswartz/.rvm/gems/ruby-2.6.3/bin/rubocop
env: ruby_executable_hooks: No such file or directory
```

So it seems that when I manually set the rubocop path in VSCode maybe it's not running in the correct context? Which is the whole reason why it's not running the right `rvm` out of the box and why I'm trying to do this method.

I tried setting the `Config File Path` setting as well but got the same error.
